packages/terra-navigation-side-menu/translations/es-ES.json

```diff
@@ -1,3 +1,5 @@
 {
-  "Terra.navigation.side.menu.selected": "Seleccionado"
+  "Terra.navigation.side.menu.selected": "Seleccionado",
+  "Terra.navigation.side.menu.folder": "Folder",
+  "Terra.navigation.side.menu.document": "Document"
```

Comment:
I think you should log translations

Reply:
yeah we need to log, will check with UX team once on this. Thanks